Makefile

```diff
@@ -100,4 +100,6 @@ clean: clean-rs clean-patch clean-html
 	@mkdir -p .deps
 	python3 scripts/makedeps.py $< > $@
 
--include $(patsubst devices/%.yaml, .deps/%.d, $(YAMLS))
+$(PATCHED_SVDS): $(patsubst devices/%.yaml, .deps/%.d, $(YAMLS))
```

Comment:
Could we stick the `.deps/%.d` dependency on L37 with `svd/%.svd.patched` instead?

Reply:
Hmm, this way make treats dep files as intermediate and removes them after build.

Reply:
I did a quick test and that seems to work great, ensuring each SVD only depends on its on `.d` file too.

Reply:
Ah, you're right about the intermediate business. Maybe stick them in .PRECIOUS too?

Reply:
Fixed!